Description: Adds the missing bits to the modello configuration that should be inherited from the ignored parent poms
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -279,6 +279,7 @@
         <version>1.8.3</version>
         <configuration>
           <version>${mdoVersion}</version>
+          <useJava5>true</useJava5>
           <models>
             <model>src/main/mdo/assembly.mdo</model>
             <model>src/main/mdo/assembly-component.mdo</model>
